src/components/Documents/DocumentTable.jsx

```diff
@@ -1,22 +1,41 @@
 // React Imports
 import React, { useContext } from 'react';
+
```

Comment:
Don't really care either way, but don't need these spacer lines between each category of imports. They don't bother me though.